### PR TITLE
Set initial cookies

### PR DESCRIPTION
**Pull request by sausame**

### CODE REVIEW

**Comment by ssbarnea** [comment]
Can you please explain the purpose of this change, preferably referencing a ticket.

**Comment by mitsuo0114** [comment]
I guess he want to similar thing as https://github.com/pycontribs/jira/pull/413

I'm in same boat. I want to save/load cookies from previous sessions. (OAuth is disabled in our JIRA at work)

File: jira/client.py [hunk 1]
```diff
@@ -323,6 +323,12 @@ def __init__(self, server=None, options=None, basic_auth=None, oauth=None, jwt=N
 
         self._session.max_retries = max_retries
 
+        # Set initial cookies for the session
+        if cookies is not None and type(cookies) is list:
```
Comment (dbaxa) [inline jira/client.py]: You could use `isinstance` instead of `type`.

**Comment by tubaman** [comment]
I like this idea better than my #413 because it doesn't assume where the cookies are coming from.

**Comment by ericfrederich** [comment]
No need for a for loop.  Just do:
```python
if cookies is not None:
    self._session.cookies.update(cookies)
```

**Comment by hdost** [comment]
@sausame Does  #484  solve the root issue?

**Comment by hdost** [comment]
Closing as develop is no longer used. 
Please feel free to rebase off master and submit a new PR.

**Pull request closed without merging**